src/components/Cellar/InventoryTabPanel.js

```diff
@@ -0,0 +1,71 @@
+/** @typedef {import("../../index").farmhand.keg} keg */
+import React, { useContext } from 'react'
+import { number } from 'prop-types'
+import Divider from '@material-ui/core/Divider'
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import ReactMarkdown from 'react-markdown'
+
+import FarmhandContext from '../Farmhand/Farmhand.context'
+import { KEG_INTEREST_RATE, PURCHASEABLE_CELLARS } from '../../constants'
+
+import { integerString } from '../../utils'
+
+import { TabPanel } from './TabPanel'
+import { Keg } from './Keg'
+
+/**
+ * @param {Object} props
+ * @param {number} props.index
+ * @param {number} props.currentTab
+ */
+export const InventoryTabPanel = ({ index, currentTab }) => {
```

Comment:
this component name is tripping me up a little. "inventory tab panel" feels like something generic, but this one here is specific to the cellar right? should it be `CellarTabPanel`?